#!/usr/bin/env python
PACKAGE = "depthvista_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("video_dev_id",		int_t,		0,	"Video device node index",			2,	0,	255);
gen.add("range",			int_t,		0,	"Depth range",						0,	0,	1);
gen.add("enable_depth",		bool_t,		0,	"Camera mode - depth required",		True);
gen.add("enable_rgb",		bool_t,		0,	"Camera mode - rgb required",		False);
gen.add("enable_ir",		bool_t,		0,	"Camera mode - ir required",		False);

exit(gen.generate(PACKAGE, "depthvista_camera", "DepthVista"))
